backend-api/tests/api-tests/utils/apiInstance.ts

```diff
@@ -4,33 +4,30 @@ import "dotenv/config";
 import { aws4Interceptor } from "aws4-axios";
 import { fromNodeProviderChain } from "@aws-sdk/credential-providers";
 
-function getInstance(baseUrl: string) {
+function getInstance(baseUrl: string, isProxyApi: boolean = false) {
```

Comment:
I think this might just be personal preference, but I don't think the getInstance function should know about the ProxyApi. 

What about...

```suggestion
function getInstance(baseUrl: string, useAwsSigv4Signing: boolean = false) {
```

Reply:
I'd also be keen on the boolean (and the baseUrl if you desire) going into an object. 

I'd much rather see...

`const apiInstance = getInstance({ baseUrl: 'https://something.com', useAwsSigv4Signing: true}`